src/store/messaging.store.ts

```diff
@@ -162,10 +169,16 @@ export const useMessagingStore = create<MessagingState>((set, g) => {
 
```

Comment:
todo: payment & message should also use the "after" parameter to optimize historical data loading